readme.md

```diff
@@ -32,6 +32,10 @@ console.log(fallbackSymbols.tick);
 console.log(replaceSymbols('✔︎ check'));
 // On terminals with Unicode symbols:  ✔︎ check
 // On other terminals:                 √ check
+
+console.log(replaceSymbols('✔︎ check', {useFallback: true}));
+// On terminals with Unicode symbols:  √︎ check
+// On other terminals:                 √ check
```

Comment:
Shouldn't these two show the same symbol since it's using the fallback no matter what?

Reply:
Those two lines use the same symbol (the fallback one).